geonode/settings.py

```diff
@@ -1131,10 +1154,6 @@
 # 2. Creating a downstream project, if you are doing a lot of customization.
 # 3. Override settings in a local_settings.py file, legacy.
 # Load more settings from a file called local_settings.py if it exists
-try:
```

Comment:
Are you dropping local_settings?

Reply:
My bad, sorry. I have several settings depending on branch, this one was causing wrong settings imported.